internal/provider/networking/network_ip_interface_resource.go

```diff
@@ -121,6 +122,18 @@ func (r *IPInterfaceResource) Schema(ctx context.Context, req resource.SchemaReq
 				},
 				Required: true,
 			},
+			"service_policy": schema.StringAttribute{
+				MarkdownDescription: "IPInterface service policy",
+				Optional:            true,
+				Computed:            true,
+				/*
+				 * Default values:
+				 *   "default-data-files" if scope is svm
+				 *   "default-management" if scope is cluster and IPspace is not Cluster (not yet implemented)
+				 *   "default-cluster" if scope is cluster and IPspace is Cluster (not yet implemented)
+				 */
+				Default: stringdefault.StaticString("default-data-files"),
```

Comment:
In ONTAP 9.6 API document:
If not specified in POST, the following default property values are assigned:
:
service_policy:
default-data-files if scope is svm
default-management if scope is cluster and IPspace is not Cluster
default-cluster if scope is svm and IPspace is Cluster
The default value can be found in query API. It should not be set here.

Reply:
Hello @chuyich, thank you very much for reviewing this PR.

I did add a default value for the following reason: without a default value, when there is no configuration supplied, Terraform will mark the attribute as unknown. Unknown attributes are constantly displayed as "(known after apply)" in plan outputs... the default value is to reduce these "(known after apply)" plan outputs.

I'm happy to remove the default value, though - just wanted to double-check that the behavior mentioned above is indeed desired in this case.

Thanks much for a quick confirmation!

Reply:
Since it could be one of 3 values based on the scope, set any of them may not be correct. It's better to use this way:
https://github.com/NetApp/terraform-provider-netapp-ontap/blob/integration/main/internal/provider/name_services/name_services_ldap_resource.go#L106
`				PlanModifiers: []planmodifier.String{
					stringplanmodifier.UseStateForUnknown(),
				},`
We found setting default values may cause issues in some cases. The above will cover the case you mentioned. Also after you remove the default value part, the parameter handling needs to be adjusted accordingly. You may use the above case as a reference.

Reply:
Understood. I've removed the default value, and added logic for fetching the value computed by ONTAP API instead.